.gitmodules

```diff
@@ -64,3 +64,12 @@
 	branch = main
 	ignore = dirty
 	shallow = true
+[submodule "externals/RmlUi"]
+	path = externals/RmlUi
+	url = https://github.com/mikke89/RmlUI
+[submodule "externals/freetype"]
+	path = externals/freetype
+	url = https://gitlab.freedesktop.org/freetype/freetype
+[submodule "externals/rlottie"]
+	path = externals/rlottie
+	url = https://github.com/Samsung/rlottie
```

Comment:
Could you add some additional properties?
```
	branch = main
	ignore = dirty
	shallow = true
```
Helps with local management and performance when checking out code.

Reply:
Done